package.json

```diff
@@ -2,7 +2,7 @@
   "name": "fabric8-analytics",
   "displayName": "Red Hat Dependency Analytics",
   "description": "Provides insights on security vulnerabilities in your application dependencies.",
-  "version": "0.10.0",
+  "version": "0.9.6",
```

Comment:
the development version is 0.10.0

Reply:
main branch should contain the currently released version, not the next version, hence reverting this to 0.9.6. https://code.visualstudio.com/api/working-with-extensions/publishing-extension#autoincrement-the-extension-version

> The stage published version should be 0.10.0-ea.x

We dont know whether the next published version will be a minor release or a patch release. The safer baseline is to expect it to be a patch version (all early-release versions will be x.y.z+1) and if we decide to do a minor release instead the continuity will still be relatively correct.